### PR TITLE
[WIP] Added Slave Database Connection Information

### LINKED ISSUE #6067: Feedback on ece-tools package topic
# General issue

I can’t find anywhere in devdocs that it says to do a DB query against a slave database in cloud you should hit port 3304 instead of 3306. 

What is not well documented is that the load balancer provides different ports for accessing MySQL for different purposes. The default port of 3304 gets spread across all 3 nodes of the cluster. while 3306 gets sent to just the non-master MySQL nodes. I’m further speculating that there’s also a port which gets you only the MySQL master.

https://devdocs.magento.com/cloud/reference/ece-tools-reference.html

## Description:

Provide information that describes the database ports for Cloud Staging and Production environments, and when you would use each one.

### PR DESCRIPTION
**Pull request by BarnyShergold**
## Purpose of this pull request

This pull request (PR) fixes #6067 

## Affected DevDocs pages
https://devdocs.magento.com/cloud/project/services-mysql.html

### CODE REVIEW

**Comment by devops-devdocs** [comment]
An admin must run tests on this PR before it can be merged.

**Comment by BarnyShergold** [comment]
@meker12  & @hguthrie - I thought I'd get this issue underway and start with some basic information. Where shall we go from here?

**Review by rogyar — APPROVED** [review]
In the future, it would be great to provide some information on how the fact of different ports affects the configuration (configuration examples)

**Comment by BarnyShergold** [comment]
> In the future, it would be great to provide some information on how the fact of different ports affects the configuration (configuration examples)

@rogyar - I agree but where can we get the definitive port information from? (BTW I wasn't expecting this PR to be reviewed yet, was waiting to discuss and expand with @hguthrie & @meker12 )

**Comment by dobooth** [comment]
Slave isn't going to work as a term. Let's use 'secondary'.

**Comment by meker12** [comment]
> Slave isn't going to work as a term. Let's use 'secondary'.
We can use that in the text, but the Magento code base still uses master/slave for databases.

File: src/cloud/project/services-mysql.md [hunk 1]
```diff
@@ -160,3 +160,9 @@ Accessing the MariaDB database directly requires you to use a SSH to log in to t
       ```bash
       mysql -h<db> -P<number> -u<username> -p<password>
       ```
+
+## Slave Database connection
+
+{:.bs-callout-info}
+If there is a requirement to connect to the Slave database, then the port *3304* should be used. For full information on
+accessing the Slave database, consult the [Magento Commerce Cloud best practice for slave connection](https://support.magento.com/hc/en-us/articles/360049045351) article.
```
Comment (meker12) [inline src/cloud/project/services-mysql.md:168]:
```suggestion
## Connect to secondary database

In some cases, you might need to connect to the secondary database, for example to improve database performance or resolve database locking issues. If this configuration is required, use port **3304** to establish the connection. See the [Magento Commerce Cloud best practice for slave connection](https://support.magento.com/hc/en-us/articles/360049045351) article in the _Magento Help Center_].   
```

**Review by meker12 — CHANGES_REQUESTED** [review]

**Review by meker12 — APPROVED** [review]

**Comment by meker12** [comment]
running tests

**Pull request merged**